### ISSUE TITLE
[Feature Request] Quicker console-based config

### ISSUE THREAD

**Issue by shocker1185** [RainbowMiner/RainbowMiner#1343]
While RB-Miner is running, I often want to change 1 or 2 settings. 
This process goes: press C (and hope that it sync's with the cycle properly)
Wait for cycle
Press S
Go through a whole bunch of settings that aren't being changed, and change the one that needs to be.

Can we bind the one-key commands to a combo like alt+c, and allow full text input? This is what I envision
set pools poolname(s) or +poolname or -poolname
set algorithm algoname(s) or +algoname or -algoname
set hashrate minername algoname hashrate
exclude algorithm....etc...

**Comment by RainbowMiner** [comment]
I planned to shift every setting to the localhost:4000. Some of the settings are already on the setup page. But one-key commands have a certain beauty. At least for me :)

**Comment by shocker1185** [comment]
> I planned to shift every setting to the localhost:4000. Some of the settings are already on the setup page. But one-key commands have a certain beauty. At least for me :)

I agree with the beauty! alt+c could also bring up the command input and retain your 1-key-awesomeness!
